### DISCUSSION TITLE
Modal Dialog should display half of the screen in nativescript angular

### DISCUSSION THREAD

**Discussion by Bahtiyar89** [NativeScript/NativeScript#9652] [Q&A]
I want to display on  Nativescript angular on pressed Modal should appear from bottom to top for display 90% of screen are there any İdeas ?

**Comment by rigor789** [comment]
You can play around with the new `RootLayout` ([docs](https://docs.nativescript.org/ui-and-styling.html#rootlayout)), or use a plugin like:
- https://github.com/nativescript-community/ui-persistent-bottomsheet
- https://github.com/nativescript-community/ui-material-components/tree/master/packages/bottomsheet

Or if only targeting iOS, you can use the standard modal, and switch it's style to use one of the available ones: https://developer.apple.com/documentation/uikit/uimodalpresentationstyle?language=objc

For example, you can pass the following to the modal options
```js
{
  ios: {
    presentationStyle: UIModalPresentationStyle.UIModalPresentationPageSheet
  }
}
```
